### ISSUE TITLE
documentation: maybe we should remove mentions to "data" folder?

### ISSUE THREAD

**Issue by villares** [py5coding/py5generator#298]
I was looking at the `create_font()` reference and then I did a quick search: https://py5coding.org/search.html?q=%27%22data%22+folder%27#

Unless I'm mistaken, in py5 you don't need a "data" folder by default, right?

**Comment by hx2A** [comment]
I would expect this to be the same as it is for Processing...is it not? I am pretty sure the data folder is required for loading shaders. There might be some places where it is mentioned but isn't actually required though.

**Comment by villares** [comment]
Hmmm, I run a quick test with `create_font()` and it looks like it can find the font both inside the "data" folder and on the same folder as the sketch. I was surprised, I assumed that in py5 things would only work like "standard" Python...

**Comment by hx2A** [comment]
Does that work for Processing also? Can Processing's `create_font()` find a font in the data folder and in the data folder's parent?

**Comment by villares** [comment]
And the answer was also a surprise for me! It works both ways!

![image](https://github.com/py5coding/py5generator/assets/3694604/6fa47a2f-7f1a-4212-b1a7-2907af197370)

**Comment by hx2A** [comment]
> And the answer was also a surprise for me! It works both ways!

Great! That's a happy surprise.

Is there anything that needs to be done here? Can we close this issue?

**Comment by villares** [comment]
yeah, I think we can close this, thanks for the insightful questions!
"hx2A, the Socratic maintainer"
